qiskit/visualization/matplotlib.py

```diff
@@ -318,8 +324,7 @@ def _get_text_width(self, text, fontsize, param=False):
                     sum_text += self._char_list['c'][f]
             return sum_text
 
-    def param_parse(self, v):
-        # create an empty list to store the parameters in
+    def _param_parse(self, v):
```

Comment:
Does the previous docstring still apply?
```suggestion
    def _param_parse(self, v):
    # create an empty list to store the parameters in
```

Reply:
Not a docstring. Just a comment about the first line of the method. Seemed superfluous.